ChangeLog

```diff
@@ -22,6 +22,8 @@ What's New in Pylint 2.7.1?
 
   Closes #4119
 
+* Workflow and packaging improvements
```

Comment:
Not in the right place anymore following the rebase :)

Reply:
The rebase worked to flawless it seems. Where shall I put it, `2.7.2` or `2.8.0`. I could split this MR up, too if you like.

Reply:
I think 2.8, I don't think we should cherry-pick too much on 2.7.